### PR TITLE
add interactive script ahead of covid-19 demo 

### PR DESCRIPTION
**Pull request by ehigham**

### CODE REVIEW

File: api/src/wfl/service/firecloud.clj [hunk 1]
```diff
@@ -44,11 +44,25 @@
                                           body-override)
                          :escape-slash false)}
          http/request
-         util/response-body-json
-         :submissionId)))
+         util/response-body-json)))
   ([workspace methodconfig entity]
    (create-submission workspace methodconfig entity {})))
```
Comment (tbl3rd) [inline api/src/wfl/service/firecloud.clj:49]: Thanks.

File: api/src/wfl/service/firecloud.clj [hunk 2]
```diff
@@ -44,11 +44,25 @@
                                           body-override)
                          :escape-slash false)}
          http/request
-         util/response-body-json
-         :submissionId)))
+         util/response-body-json)))
```
Comment (okotsopoulos) [inline api/src/wfl/service/firecloud.clj:47]: I know you probably made this and similar changes to get things to work, but I like this approach.  Let the invocation decide if it wants to toss the rest of the response.

File: api/test/wfl/system/cdc_covid19_surveillance_demo.clj [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+(defn delete-workspace [workspace]
+  (println "Deleting" workspace)
+  (firecloud/delete-workspace workspace))
+
+(defn demo []
+  (println "CDC COVID-19 Surveillance Demo")
+  (println "Demonstrating sarscov2_illimina_full automation")
+  (wait-for-user)
+  (fixtures/with-fixtures
+    [(util/bracket clone-workspace delete-workspace)
```
Comment (okotsopoulos) [inline api/test/wfl/system/cdc_covid19_surveillance_demo.clj:100]: I'm guessing that if we delete a workspace, everything within it (snapshot references, methodconfigs, submissions) are also deleted.

**Review by okotsopoulos — APPROVED** [review]
When the dust settles on this demo, I hope we keep this script around.  Some useful patterns for future demos and testing.

I see a rawls test failing but think that's on your radar already.

File: api/test/wfl/system/cdc_covid19_surveillance_demo.clj (hunk 3 above)
Comment (ehigham) [reply api/test/wfl/system/cdc_covid19_surveillance_demo.clj:101]: Yep

**Comment by ehigham** [comment]
> When the dust settles on this demo, I hope we keep this script around. Some useful patterns for future demos and testing.
> 
> I see a rawls test failing but think that's on your radar already.
Yeah. The method-configuration changed upstream. I've asked Christine to tweak it for us.

File: api/src/wfl/service/firecloud.clj [hunk 4]
```diff
@@ -44,11 +44,25 @@
...
+  (let [[mcns mcn] (str/split methodconfig #"/")]
+    (-> (workspace-api-url workspace "submissions")
+        (http/post {:headers      (auth/get-auth-header)
+                    :content-type :application/json
+                    :body         (json/write-str
+                                   {:methodConfigurationNamespace mcns
+                                    :methodConfigurationName mcn
+                                    :useCallCache true}
+                                   :escape-slash false)})
+        util/response-body-json)))
```
[the remark this answers is not in the document]
Comment (ehigham) [reply api/src/wfl/service/firecloud.clj:62]: Thanks

**Pull request merged**